payroll/views/hr_salary_rule_views.xml

```diff
@@ -192,6 +214,108 @@
                             </tree>
                         </field>
                     </page>
+                    <page name="help" string="Help">
+                        <group>
+                            <div style="margin-top: 4px;">
+                                <h3>Salary Rules formula definition (Python)</h3>
+                                <p>
+                                    In python definition in salary rules, you can write any code like you will do
+                                    in python. Any methematical operation or python function is supported.
```

Comment:
s/methematical/mathematical/

Reply:
Fixed the repeated words in last commit, thanks.